packages/foreman/foreman/foreman.spec

```diff
@@ -711,6 +713,43 @@ Meta Package to install requirements for Redis caching support
 %files redis
 %{_datadir}/%{name}/bundler.d/redis.rb
 
+%package dynflow-sidekiq
+Summary: Foreman Redis caching support
+Group:  Applications/System
+# start specfile redis Requires
+Requires: %{?scl_prefix}rubygem(sidekiq) >= 5.0
+Requires: %{?scl_prefix}rubygem(sidekiq) < 6
+Requires: %{?scl_prefix}rubygem(gitlab-sidekiq-fetcher)
+# end specfile redis Requires
+Requires: %{name} = %{version}-%{release}
+
+%description dynflow-sidekiq
+Meta Package to install dynflow sidekiq executor support
+
+%files dynflow-sidekiq
+%{_datadir}/%{name}/bundler.d/sidekiq.rb
+%{_unitdir}/%{dynflow_sidekiq_service_name}.service
+%{_datadir}/%{name}/script/dynflow-sidekiq.rb
+
+%install dynflow-sidekiq
```

Comment:
I don't think you can have separate `%install` sections. At least, we don't use it anywhere else.

Reply:
So I should install it in the main package directly? or to use build section?

Reply:
Main `%install` is fine. Then within `%files` you split it up again.